### ISSUE TITLE
Investigate a solution for working with KDBX3 and KDBX4

### ISSUE THREAD

**Issue by haroon-sheikh** [sitture/env-config#89]
# Problem

EnvConfig is currently using the library below for working with keepass databases, however the library is no longer maintained and deprecated.  

```xml
<dependency>
  <groupId>de.slackspace</groupId>
  <artifactId>openkeepass</artifactId>
  <version>${openkeepass.version}</version>
</dependency>
```

# Tasks

- Investigate if there's another library that can be replaced with
- Should be able to work with both KDBX3 and KDBX4 versions.

**Comment by haroon-sheikh** [comment]
Potentially can migrate to https://github.com/jorabin/KeePassJava2

**Comment by haroon-sheikh** [comment]
Fixed in version 0.11.0.
